spec/application/init.js

```diff
@@ -0,0 +1,6 @@
+const Validator = require('is-my-ssb-valid')
+const schema = require('private-group-spec').schema.application.init
```

Comment:
I'd like this schema to be in the ssb-tribes repo

Reply:
Reference : https://github.com/ssbc/private-group-spec/pull/10/files ?

Reply:
There's no init method. This should just be removed, correct?